subprojects/docs/src/samples/userguide/dependencyManagement/workingWithDependencies/iterateArtifacts/kotlin/build.gradle.kts

```diff
@@ -0,0 +1,23 @@
+repositories {
+    jcenter()
+}
+
+val scm by configurations.creating
+
+dependencies {
+    scm("org.eclipse.jgit:org.eclipse.jgit:4.9.2.201712150930-r")
+    scm("commons-codec:commons-codec:1.7")
+}
+
+// tag::iteration-task[]
+task("iterateResolvedArtifacts") {
+    val scm = configurations["scm"]
```

Comment:
:x: `val scm by configurations`

Reply:
I'll pass on that one. @eriwen just told me to avoid delegated properties to stay close to the groovy sample in the user documentation.